message_definitions/v1.0/common.xml

```diff
@@ -3913,11 +3913,11 @@
       <description>Estimator status message including flags, innovation test ratios and estimated accuracies. The flags message is an integer bitmask containing information on which EKF outputs are valid. See the ESTIMATOR_STATUS_FLAGS enum definition for further information. The innovaton test ratios show the magnitude of the sensor innovation divided by the innovation check threshold. Under normal operation the innovaton test ratios should be below 0.5 with occasional values up to 1.0. Values greater than 1.0 should be rare under normal operation and indicate that a measurement has been rejected by the filter. The user should be notified if an innovation test ratio greater than 1.0 is recorded. Notifications for values in the range between 0.5 and 1.0 should be optional and controllable by the user.</description>
```

Comment:
Good catch from @tridge and @WickedShell the unit here is not Hz
https://github.com/PX4/ecl/blob/cdbca91e79fad33a8908c009d66f14a182d55f1e/EKF/ekf_helper.cpp#L1105